src/perf/lib/Tcp.cpp

```diff
@@ -199,7 +199,7 @@ bool TcpWorker::Initialize(TcpEngine* _Engine, uint16_t PartitionIndex)
     Engine = _Engine;
     ExecutionContext.Callback = DoWork;
     ExecutionContext.Context = this;
-    ExecutionContext.Ready = TRUE;
+    WriteBooleanNoFence(&ExecutionContext.Ready, TRUE);
```

Comment:
I've reverted back to `InterlockedFetchAndSetBoolean` for now since it's cross platform. We can look at a future solution for cross-platform `WriteBooleanNoFence`.